docs/partials/developer-docs.rst

```diff
@@ -0,0 +1,15 @@
+.. note::
+
+    This documentation page is aimed at developers who want to customize,
+    change or extend the code of OpenWISP WiFi Login Pages in order to
+    modify its behavior (eg: for personal or commercial purposes or to fix
+    a bug, implement a new feature or contribute to the project in
+    general).
+
+    If you aren't a developer and you are looking for information on how
+    to use OpenWISP, please refer to:
+
+    - :doc:`General OpenWISP Quickstart </user/quickstart>`
```

Comment:
I am not sure this is relevant here... what do you think? We could remove it and leave only the following 2 items.